shared/util.js

```diff
@@ -308,12 +324,13 @@ class Util {
 
   /**
    * Returns a primary domain for provided hostname (e.g. www.example.com -> example.com).
+   * This function is used only while rendering pre-10.0 LHRs.
```

Comment:
```suggestion
   * As it doesn't consult the Public Suffix List, it can sometimes lose detail.
   * See the `listOfTlds` comment above for more.
   * This function is used only while rendering pre-10.0 LHRs.
```